packages/ui/src/ValueView/index.tsx

```diff
@@ -46,14 +60,24 @@ export const ValueViewComponent = <SelectedContext extends Context = 'default'>(
   valueView,
   context,
   priority = 'primary',
+  hideIcon = false,
+  hideSymbol = false,
+  abbreviate = false,
 }: ValueViewComponentProps<SelectedContext>) => {
   const density = useDensity();
 
   if (!valueView) {
     return null;
   }
 
-  const formattedAmount = getFormattedAmtFromValueView(valueView, true);
+  let formattedAmount: string;
+  if (abbreviate) {
+    const amount = getFormattedAmtFromValueView(valueView, false);
```

Comment:
comment: what if `getFormattedAmtFromValueView` returns something unparseable (eg. null, undefined)? should we add validation guards to shortify to prevent breakage?

Reply:
The function returns a `string` type, so all good.